django_components/component.py

```diff
@@ -173,7 +173,7 @@ def render_to_response(
     def _fill_slots(
         self,
         slots_data: Dict[SlotName, str],
-        escape_content: bool,
+        escape_content: Union[bool, None],
```

Comment:
I saw this and realized that I got the `escape_slots_content` type wrong in `render_to_response`. I think that one should be:

```
    def render_to_response(
        self,
        context_data: Dict[str, Any],
        slots_data: Optional[Dict[SlotName, str]] = None,
        escape_slots_content: bool = True,
        *args,
        **kwargs,
    ):
```

And this one should be only `bool`, not `None`.

Reply:
Fixed in separate commit.